src/components/CustomCard/CustomCard.tsx

```diff
@@ -31,9 +34,14 @@ export const CustomCard = ({
     <Pressable
       onPress={() => {
```

Comment:
any event handlers other than something trivial are good separated into a function, eg `handleCardPress`, also not a fan of this nested selectedBedId && within the ternary